### PR TITLE
WSTEAM1-313: Clean up embeds following BFF changes

### PR DESCRIPTION
**Pull request by Isabella-Mitchell**
Resolves JIRA subtask [WSTEAM1-704](https://jira.dev.bbc.co.uk/browse/WSTEAM1-704) of [WSTEAM1-313](https://jira.dev.bbc.co.uk/browse/WSTEAM1-313) - Embeds: Optimo Telescope embeds (.amp support only).

Also adds tests for [WSTEAM1-437](https://jira.dev.bbc.co.uk/browse/WSTEAM1-437) - Embeds: Optimo Sport embeds (.amp support only)

This will be merged once the correlating BFF PR has been merged and deployed to live

Overall changes
======
Cleans up codebase following BFF changes. Refactors and adds unit tests.

Code changes
======
- Update fixture data to match modified BFF response
- Remove unnecessary code from the UnsupportedEmbed component now that it does not also handle the Uploader
- Refactor unit tests to use it.each and check over Telescope, Player Rater and Team Selector fixture data (This could have been in a seperate PR - though I didn't want to raise another PR just to do this).

Testing
======
1. Testing on test as past of (https://jira.dev.bbc.co.uk/browse/WSTEAM1-313) - Embeds: Optimo Telescope embeds (.amp support only).

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/pages/ArticlePage/ArticlePage.jsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ const ArticlePage = ({ pageData }) => {
     timestamp: props =>
       hasByline ? null : <Timestamp {...props} popOut={false} />,
     social: SocialEmbedContainer,
-    embed: props => <UnsupportedEmbed {...props} />,
+    embed: UnsupportedEmbed,
```
Comment (amoore108) [inline src/app/pages/ArticlePage/ArticlePage.jsx:139]: Nice 👍 Should we apply this approach to the other embed types below for consistency?

**Review by amoore108 — APPROVED** [review]
Looks great, like the approach taken here. Just a wee comment on the prop passing in the ArticlePage component.

File: src/app/pages/ArticlePage/ArticlePage.jsx (hunk 1 above)
Comment (Isabella-Mitchell) [reply src/app/pages/ArticlePage/ArticlePage.jsx:139]: Oh right, yes have done for embeds. I didn't realise these were unnecessary.

Comment (amoore108) [reply src/app/pages/ArticlePage/ArticlePage.jsx:139]: Yea its a bit of a weird syntax. The `props` will just get automatically passed through to the component when defined this way. Only when you need to pass in something specific or extra, like `popOut={false}` above, then you'd need to specify the overall `props` to spread in.

**Review by shayneahchoon — APPROVED** [review]

**Pull request merged**